### PR TITLE
DOC add 3D output shape description

### PR DESCRIPTION
**Pull request by eickenberg**

### CODE REVIEW

File: doc/source/userguide.rst [hunk 1]
```diff
@@ -120,7 +120,15 @@ size:
 3-D
 ---
 
-...
+For an input array of shape $(B, C, N_1, N_2, N_3)$, a Solid Harmonic Scattering with $J$
+scales and $L$ angular frequencies, which applies $P$ different types of $\mathcal L_p$ 
+spatial averaging will result in an output of shape
```
Comment (edouardoyallon) [inline doc/source/userguide.rst]: 2 points?

**Comment by janden** [comment]
Or “scattering,” for that matter.

**Comment by edouardoyallon** [comment]
btw since when is the scattering transform a feature extractor

**Comment by eickenberg** [comment]
> btw since when is the scattering transform a feature extractor

where is this stated?
And why would this be false?
What is it instead?

**Comment by eickenberg** [comment]
Changed the capitalization on both changes I made

**Comment by janden** [comment]
> Changed the capitalization on both changes I made

Thanks.

File: doc/source/userguide.rst (hunk 1 above)
Comment (eickenberg) [reply doc/source/userguide.rst]: What does that mean?

Comment (edouardoyallon) [reply doc/source/userguide.rst]: shape <-> shape:  ?

File: doc/source/userguide.rst [hunk 2]
```diff
@@ -120,7 +120,15 @@ size:
...
+
+$(B, C, L, J + J(J - 1) / 2, P).$
+
```
Comment (edouardoyallon) [inline doc/source/userguide.rst]: $(B, C, L, J + J(J - 1) / 2, P).$
 <->
$(B, C, L, J + J(J - 1) / 2, P)\,.$

Comment (eickenberg) [reply doc/source/userguide.rst]: `.$` to `,.$`?

File: doc/source/userguide.rst (hunk 1 above)
Comment (janden) [reply doc/source/userguide.rst]: Francisms...

Comment (janden) [reply doc/source/userguide.rst]: In English, that's a “colon”.

File: doc/source/userguide.rst (hunk 2 above)
Comment (edouardoyallon) [reply doc/source/userguide.rst]: \, sorry, just an insecable space between a point and the paranthesis

File: doc/source/userguide.rst (hunk 1 above)
Comment (edouardoyallon) [reply doc/source/userguide.rst]: oh oops 🐶

File: doc/source/userguide.rst (hunk 2 above)
Comment (janden) [reply doc/source/userguide.rst]: So that's `~.$` instead of `.$`?

Comment (janden) [reply doc/source/userguide.rst]: (Called a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space).)

Comment (edouardoyallon) [reply doc/source/userguide.rst]: damned, github parses my stuff, yes "vague" or "backslash virgule" is ok for me

Comment (edouardoyallon) [reply doc/source/userguide.rst]: yeah a https://fr.wikipedia.org/wiki/Espace_ins%C3%A9cable

**Comment by janden** [comment]
# [Codecov](https://codecov.io/gh/kymatio/kymatio/pull/592?src=pr&el=h1) Report
> Merging [#592](https://codecov.io/gh/kymatio/kymatio/pull/592?src=pr&el=desc) into [dev](https://codecov.io/gh/kymatio/kymatio/commit/7d40c99e3810d792befec94a2d11854b472b750f&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/kymatio/kymatio/pull/592?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##              dev     #592   +/-   ##
=======================================
  Coverage   87.70%   87.70%           
=======================================
  Files          65       65           
  Lines        2342     2342           
=======================================
  Hits         2054     2054           
  Misses        288      288           
```

| Flag | Coverage Δ | |
|---|---|---|
| #jenkins_main | `87.70% <ø> (ø)` | |
| #travis | `77.79% <ø> (ø)` | |


------

[Continue to review full report at Codecov](https://codecov.io/gh/kymatio/kymatio/pull/592?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/kymatio/kymatio/pull/592?src=pr&el=footer). Last update [7d40c99...22d5884](https://codecov.io/gh/kymatio/kymatio/pull/592?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: doc/source/userguide.rst (hunk 1 above)
Comment (eickenberg) [reply doc/source/userguide.rst]: I disagree with putting a colon here. It's a math sentence, just with a newline.

File: doc/source/userguide.rst (hunk 2 above)
Comment (eickenberg) [reply doc/source/userguide.rst]: but this is English. Non insecable spaces like that in English. I might still add one to make you happy though.

Comment (edouardoyallon) [reply doc/source/userguide.rst]: ok sorry non non

Comment (eickenberg) [reply doc/source/userguide.rst]: I fixed a bug and added an inescapable space

Comment (edouardoyallon) [reply doc/source/userguide.rst]: i'm pros merging but maybe u don't like non-break space(i thought they should always surround equations)

Comment (eickenberg) [reply doc/source/userguide.rst]: now I like it. Let's merge

**Pull request merged**